plugins/AccountStats.py

```diff
@@ -78,9 +82,9 @@ def get_db_version(self):
         return self.db.execute("PRAGMA user_version").fetchone()[0]
 
     def fetch_history(self, first_time_stamp, last_time_stamp):
-        history = self.api.return_lending_history(first_time_stamp, last_time_stamp - 1, 50000)
+        history = self.api.return_lending_history(first_time_stamp, last_time_stamp - 1, 5000)
         loans = []
-        for loan in reversed(history):
```

Comment:
Didn't see the need to waste the CPU in reversing this array. It gets ordered anyway within the database.